huaweicloud/services/waf/resource_huaweicloud_waf_dedicated_instance.go

```diff
@@ -105,6 +105,10 @@ func ResourceWafDedicatedInstance() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
 
```

Comment:
```suggestion
			"enterprise_project_id": {
				Type:          schema.TypeString,
				Optional:    true,
				Computed: true,
			},
```

Reply:
Set `enterprise_project_id` value in Read.

Reply:
We cannot get `enterprise_project_id` from read request.

Reply:
ok, I get it.